src/fiatExchanges/quotes/NormalizedQuote.ts

```diff
@@ -4,13 +4,20 @@ import { FiatExchangeEvents } from 'src/analytics/Events'
 import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
 import { SettlementTime } from 'src/fiatExchanges/quotes/constants'
 import { CICOFlow, PaymentMethod } from 'src/fiatExchanges/utils'
-import { Currency } from 'src/utils/currencies'
+import { TokenBalance } from 'src/tokens/slice'
+import { CiCoCurrency, Currency } from 'src/utils/currencies'
 
 export default abstract class NormalizedQuote {
   abstract getPaymentMethod(): PaymentMethod
-  abstract getFeeInFiat(exchangeRates: { [token in Currency]: string | null }): BigNumber | null
-  abstract getFeeInCrypto(exchangeRates: { [token in Currency]: string | null }): BigNumber | null
-  abstract getCryptoType(): Currency
+  abstract getFeeInFiat(
+    exchangeRates: { [token in Currency]: string | null },
+    tokenInfo: TokenBalance | undefined
```

Comment:
the tokenInfo param being optional here and for getFeeInCrypto confuses me. It seems like it is required to get a meaningful result almost all the time, except for getFeeInFiat on the ExternalQuote class.  I'd prefer if it was just made required and the consumers did null handling themselves, to make it less opaque what happens when you don't have a truthy tokenInfo value.